extensions/amp-story/0.1/media-pool.js

```diff
@@ -575,8 +594,17 @@ export class MediaPool {
     const isMuted = mediaEl.muted;
     const currentTime = mediaEl.currentTime;
 
-    mediaEl.muted = false;
-    return Promise.resolve(mediaEl.play()).then(() => {
+    // If the video is already playing, we do not want to call play again, as it
+    // can interrupt the video playback.  Instead, we do a no-op.
+    const playFn = () => {
+      if (isPaused) {
+        mediaEl.play();
+      }
+    };
```

Comment:
If this works sync then you can just remove `playFn` and call directly.

Reply:
It has to be wrapped in a promise because in some browsers, `play()` returns a boolean (IE), whereas other browsers `play()` returns a promise yielding a boolean (literally everything other than IE).

I could pass the anonymous function directly, but this is just pulled out for readability.

Reply:
Ugh, I see. Comment explaining this would be nice. Feel free to add in a follow-up so we can get this in.